unitTestSupportFunctions/inputInletVelocityParabolic.m

```diff
@@ -0,0 +1,56 @@
+%% Function documentation
```

Comment:
Follow my coding style. First comes the function signature, then the licensing information and then the function documentation